microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -30,7 +30,7 @@ For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS
     :::image-end:::  
     
     > [!IMPORTANT]
-    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeDarkTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  
+    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  
```

Comment:
```suggestion
    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  Select a **Rendering** command, rather than an **Appearance** command.
```